projects/client-api-react/src/index.js

```diff
@@ -531,7 +526,19 @@ const Graphistry = forwardRef((props, ref) => {
 
     useEffect(() => {
         if (g && onSelectionUpdate) {
+            let isFirst = true;
+
             const sub = selectionUpdates(g, selectionUpdateOptions)
+                // use first selection update as signal that data is synced
+                // trigger ref init
+                .pipe(
+                    tap(() => {
+                        if (isFirst) {
+                            setHasInitRef(true);
```

Comment:
can we encapsulate the fix into `client-api` , and leave react out of it, e.g., no `g` object until `client-api` is ready?